lib/EncodeImagesStage.js

```diff
@@ -4,20 +4,32 @@ var Promise = require('bluebird');
 
 Jimp.prototype.getBufferAsync = Promise.promisify(Jimp.prototype.getBuffer);
 
-module.exports = encodeImages;
+module.exports = EncodeImagesStage;
+
+/**
+ * Encodes Jimp images in pipeline extras to buffers.
+ *
+ * The glTF asset must be initialized for the pipeline.
+ */
+function EncodeImagesStage() {
+    this.name = 'encodeImages';
+    this.before = ['quantizeAttributes'];
+    this.after = [];
```

Comment:
If we are going to fill out the `before`/`after` stages in this PR, we will probably want to do the dependency graph creation and traversal as part of this PR so we know these are right.

@shehzan10 are you familiar with the plan here?

There is also some graph code in Cesium that will be of interest: [Source/Renderer/ShaderSource.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Renderer/ShaderSource.js)

Please discuss with me if you need help.